util-scripts/build-daedalus-bridge.sh

```diff
@@ -1,4 +1 @@
-./util-scripts/clean.sh
-./util-scripts/build.sh
-stack exec -- cardano-wallet-hs2purs
-cd daedalus && npm install && npm run build:prod && npm link
+./util-scripts/build-client-api.sh && npm link
```

Comment:
`npm link` creates a library and puts it into `npm` directory. From there UI is pulling the library